backend/src/main/java/com/ody/notification/service/NotificationService.java

```diff
@@ -81,11 +81,11 @@ public List<Notification> findAllMeetingLogs(Long meetingId) {
     }
 
     @Transactional
-    public void sendNudgeMessage(Mate mate) {
-        Notification notification = Notification.createNudge(mate);
+    public void sendNudgeMessage(Mate requestMate, Mate nudgedMate) {
+        Notification notification = Notification.createNudge(nudgedMate);
         Notification nudgeNotification = notificationRepository.save(notification);
-        NudgeRequest nudgeRequest = new NudgeRequest(mate, nudgeNotification);
-        fcmPushSender.sendNudgeMessage(nudgeRequest);
-        nudgeNotification.updateStatusToDone();
+        NudgeMessage nudgeMessage = new NudgeMessage(nudgedMate.getMember().getDeviceToken(),
+                requestMate.getNicknameValue());
```

Comment:
[제안]

`getMemberDeviceToken()` 만들어주는건 어떤가요?

```suggestion
        NudgeMessage nudgeMessage = new NudgeMessage(
                nudgedMate.getMemberDeviceToken(),
                requestMate.getNicknameValue()
        );
```

Reply:
이부분은 #410 PR 머지되면 반영될 예정이라
충돌 최소화하려면 메서드 명을 동일하게 맞춰야할 것 같아요